apps/prairielearn/src/sprocs/sync_assessments.sql

```diff
@@ -452,43 +452,27 @@ BEGIN
 
                     IF (valid_assessment.data->>'group_work')::boolean THEN
                         -- Iterate over all group roles in assessment
-                        FOR valid_group_role IN (
+                        WITH valid_group_roles AS (
                             SELECT gr.id, gr.role_name
                             FROM group_roles as gr
                             WHERE gr.assessment_id = new_assessment_id
```

Comment:
Nit: this CTE can very easily be dismissed and replaced with a direct select on `group_roles` in the INSERT/SELECT statement below.

Reply:
Done in f0814d7.